app/Http/Controllers/SettingController.php

```diff
@@ -2,9 +2,86 @@
 
 namespace App\Http\Controllers;
 
+use App\Http\Requests\StoreSettingRequest;
+use App\Models\Setting;
+use App\Models\User;
 use Illuminate\Http\Request;
+use Illuminate\Validation\Rule;
 
 class SettingController extends Controller
 {
-    //
+    public function index()
+    {
+        // get Sam to consolidate this, it could be improved, maybe a different way instead of get()?
+        return view('settings.index',[
+            'settings' => auth()->user()->settings()->get()
+        ]);
```

Comment:
Yeah spot on, so `get` will retrieve all results out of the database. 

You don't have to worry about making the changes now because you're using a view. But typically when we are building routes up in our API, the "show all" results are paginated, which means they will only return a sub-set of all results but also display the total, per page and the next_page to help the developer make the cool paginated buttons like:

[<] 1 2 3 4 5 [>]

You can paginate easily in Laravel with `->paginate()` at the end. 

https://laravel.com/docs/9.x/pagination#main-content

Reply:
I would also recommend using Resources when dealing with APIs as we use them on Plannr. Resources are REALLY cool. They let you format your model data to the front-end really nicely:

https://laravel.com/docs/9.x/eloquent-resources#main-content

But again, you are using views so this doesn't apply right now.

Reply:
You can also tidy this up by assigning `auth()->user()` their own variable, for example:

```php
$user = auth()->user();

return view('settings.index',[
    'settings' => $user->settings()->get()
]);
```